packages/ui/src/components/Checkbox.tsx

```diff
@@ -6,10 +6,7 @@ import {
 
 export interface CheckboxProps extends Omit<CheckboxPropsUi, 'onChange'> {
   indeterminate?: boolean;
-  onChange?: (
-    event: React.ChangeEvent<HTMLInputElement>,
-    indeterminate: boolean
-  ) => void;
+  onChange?: (event: React.ChangeEvent<HTMLInputElement>) => void;
```

Comment:
We can also remove `onChange` and the `Omit<CheckboxPropsUi, 'onChange'>`

Reply:
i removed this